arch/x86_64/rsp/rsp.c

```diff
@@ -101,29 +101,6 @@ cen64_align(static const uint16_t sll_l2b_keys[16][8], CACHE_LINE_SIZE) = {
   {0x8080, 0x8080, 0x8080, 0x8080, 0x8080, 0x8080, 0x8080, 0x0180},
 };
 
-// Shift right LUT; shifts in zeros from the left, one byte at a time.
-cen64_align(static const uint16_t srl_b2l_keys[16][8], CACHE_LINE_SIZE) = {
```

Comment:
My biggest hesitation to merging this PR is this table may be useful in the future even though it is unused now. There may be a better way to ignore the warning that this is unused.